### ISSUE TITLE
NTR: 5-hydroxyindoleacetic acid high in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8484]
New term request 5-hydroxyindoleacetic acid high in CSF (elthat_181122073223): 5-HIAA; cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000823
- label: Elevated CSF 5-hydroxyindoleacetic acid concentration
- parents: [Abnormal CSF carboxylic acid concentration (HP:0500183)](https://hpo.jax.org/app/browse/term/HP:0500183)
- pmid: 
- def: The concentration of 5-hydroxyindoleacetic acid in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8484
- synonyms:
